LIPS updates according to merge-pack upgrade

loga4 opened the pull request:
Updated LIP specifications according to merge-ready-pack 

context: https://research.lido.fi/t/announcement-merge-ready-protocol-service-pack/2184

loga4 committed:
LIP-12 updates according to merge-pack upgrade

loga4 committed:
fix typo

loga4 committed:
LIP-6 updates

loga4 committed:
LIP-7 updates

loga4 committed:
LIP-7: fix typo

loga4 committed:
LIP-9: rename  event

loga4 committed:
LIP-14: update stake limits

loga4 committed:
fix typo

loga4 committed:
fix typo

folkyatina commented on LIPS/lip-12.md:
```diff
@@ -56,68 +56,149 @@ A rewards vault is represented with the new `LidoMevTxFeeVault` contract. Reward
-The function is also renamed to better match it's new fucntionality.
+The function is also renamed to better match it's new functionality.
 
 #### Function: getTotalMevTxFeeCollected
```
Remove?

folkyatina replied:
And `getTotalMevTxFeeCollected()`?

loga4 replied:
yep, and `getMevTxFeeVault` too, ty!

folkyatina commented on LIPS/lip-14.md:
```diff
@@ -18,7 +18,8 @@ Add a staking rate limiting feature to have the soft moving cap for the stake am
-- Staking can be resumed then with a rate limit set or without rate limit at all
+- Staking can be resumed then by calling the `resumeStaking()` method 
+- Staking limit can be removed by calling the `removeStakingLimit()` method
 - The rate limit is managed by two parameters:
```
I'd rather go with three points here. 
- Staking resume/pause. 
- Staking limit set/remove
- Staking limit parameters

loga4 replied:
Agree, it looks much cleaner!

folkyatina commented on LIPS/lip-14.md:
````diff
@@ -45,41 +46,92 @@ We propose to accomplish the `Lido.sol` contract with additional functions to co
+
+#### Function: getStakeLimitFullInfo
+```solidity
+function getStakeLimitFullInfo() external view returns (
````
I think we need a description for calculated parameters here as well as a brief motivation why did we choose exactly these parameters.

loga4 replied:
As for the calculated variables, this is redundant here. There is a detailed description above, plus a discussion on the forum. According to the parameters, I added a small note about integrations)

folkyatina commented on LIPS/lip-7.md:
```diff
@@ -131,15 +132,15 @@ Iteratively calls `callback.processLidoOracleReport` for each stored `callback`
 ## Security considerations
 
-#### Upgradability
+#### Upgradeability
```
Actually, both are right. )

loga4 replied:
ok)

folkyatina commented on LIPS/lip-7.md:
```diff
@@ -131,15 +132,15 @@ Iteratively calls `callback.processLidoOracleReport` for each stored `callback`
 ## Gas effects
 
-In the case of `ComositePostRebaseBeaconReceiver` used with only one `IBeaconReportReceiver` callback set, spendings increase approximately by the ~4600 gas compared to direct usage of the underlying callback with `LidoOracle` (without composite adapter).
+In the case of `CompositePostRebaseBeaconReceiver` used with only one `IBeaconReportReceiver` callback set, spending's increase approximately by the ~4600 gas compared to direct usage of the underlying callback with `LidoOracle` (without composite adapter).
```
Replace `spending's` with `spending`?

loga4 replied:
done

folkyatina reviewed (commented):
Couple of places to improve

loga4 committed:
LIPS updates

- remove funcs
- add some notes

TheDZhon committed:
fix: update LIP-6

Add implementation mark and update the last change date.
Soften the references to Anchor.

TheDZhon committed:
fix: update LIP-7

Add implementation mark and update the last change date.

TheDZhon committed:
fix: update lip-4

Rename LIP-4 to reproduce its membership in the 'Protocol safeguards'
group.

TheDZhon committed:
fix: update LIP-8

Add implementation mark and update the last changed date.

TheDZhon committed:
fix: update lip-9

Add implementation mark and current date.

TheDZhon committed:
fix: update lip-10

Add implementation mark & current date.

TheDZhon committed:
fix: update lip-11

Add implementation mark & current date.

TheDZhon committed:
fix: update lip-14

Add implementation mark & current date

TheDZhon committed:
fix: update lip-14

TheDZhon committed:
fix: update lip-15

Add implementation mark & current date.

TheDZhon reviewed (approved):
Fasten seat belts :rocket:
